authserver/api/role.py

```diff
@@ -128,6 +127,90 @@ def update(self, id: str, partial=True):
             return self.response_handler.exception_response(exception_name, request=request_data)
 
 
+class AuthorizedScopeResource(Resource):
+    """Authorized scope resource for linking roles to scopes. """
```

Comment:
I'd prefer a most robust docstring here, since it took me second to realize why you have `AuthorizedScopeResource` in `role.py`. 

With that said, such confusion could be resolved with a different name space. (See my comment on line 357 of `class AuthorizedScope`.)